### PR TITLE
DLS-7267 CGT [Generic] [FE]: Upgrade the 'Accessibility Statement' Page

### PR DESCRIPTION
**Pull request by goodvibes**
Upgrade the 'Accessibility Statement' Page to use the new layout and govuk stndards

https://jira.tools.tax.service.gov.uk/browse/DLS-7267

### CODE REVIEW

**Review by LAJW — DISMISSED** [review]
LGTM

**Review by LAJW — APPROVED** [review]
Still LGTM after re-signing, but another front-end dev should take a look at the rendered version.

**Review by nubz — APPROVED** [review]

**Pull request merged**
